crates/top/re_sdk/src/recording_stream.rs

```diff
@@ -1859,69 +1872,91 @@ impl RecordingStream {
     ///
     /// This does **not** wait for the flush to propagate (see [`Self::flush_blocking`]).
     /// See [`RecordingStream`] docs for ordering semantics and multithreading guarantees.
-    pub fn flush_async(&self) {
+    ///
+    /// Convenience function for calling [`Self::flush`] with a `timeout` of `None`.
+    pub fn flush_async(&self) -> Result<(), SinkFlushError> {
+        re_tracing::profile_function!();
+        self.flush(None)
+    }
+
+    /// Initiates a flush the batching pipeline and waits for it to propagate.
```

Comment:
```suggestion
    /// Initiates a flush the batching pipeline and waits for it to finish.
```

or is there some deeper difference?